fuel-derive/src/compression/attribute.rs

```diff
@@ -0,0 +1,100 @@
+use proc_macro2::TokenTree as TokenTree2;
+
+use syn::parse::{
+    Parse,
+    ParseStream,
+};
+
+const ATTR: &str = "da_compress";
```

Comment:
Maybe it is better to use the name of the macro to link derive macro and its annotations?

Reply:
The same attribute is used in both `Compress` and `Decompress`. Renaming it to `compress` might be a good idea.

Reply:
Renamed to `compress` in efd2dc67